Source/Scene/PointCloud3DTileContent.js

```diff
@@ -521,6 +524,30 @@ define([
             },
             u_constantColor : function() {
                 return content._constantColor;
+            },
+            pc_numClippingPlanes : function() {
```

Comment:
Same not about naming here. For consistency, use `u_` like the other uniforms.

`u_clippingPlanesLength`